package.json

```diff
@@ -1,32 +1,31 @@
 {
```

Comment:
I ... don't think this is correct, and I suspect it's going to break things. You may want to check with @NeilFraser.

It looks like you're overwriting Blockly's package.json with the details for the angular 2 npm server. We don't need the latter in order to make use of AccessibleBlockly, right? In any case, I'm pretty sure this isn't the file to modify anyway.

If AccessibleBlockly needs dependencies and stuff like that, please include a README so that external developers know how to set it up.